### DISCUSSION TITLE
Cypress: equal to the alias

### DISCUSSION THREAD

**Discussion by MaxAllianov** [cypress-io/cypress#15432] [Questions and Help] [answered]
Hi everyone!

I guess it is an easy question, but I'm stuck on this for an hour.
How can I count the table rows and match them with the text?

Here is the code: 
![image](https://user-images.githubusercontent.com/76516974/110812642-754fd600-8290-11eb-938b-ccd5c4cbe28a.png)

I have already found out how to count the TRs but I can't deal with the matching.
Here is my code at the moment:
`cy.get('#other_thai').parent().children('table').children('tbody').find('tr').its('length').should('eq',QUESTION)`

The code is already working if to set the constant numerical value (8 in my case)
I guess I should create an alias for the "QUESTION" using the regex (to get only numerical value from the text using the \d+) but I'm not sure how to do this.

Please advise how to create such alias
There is also a question: will it actually work? Can I use the alias for the eq?

Thank you for your help!

**Comment by bahmutov** [comment, marked as answer]
Duplicate of https://github.com/cypress-io/cypress/discussions/15433
